extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.js

```diff
@@ -656,9 +654,13 @@ export class AmpStoryDevToolsTabPreview extends AMP.BaseElement {
    * @private
    * */
   repositionDevices_() {
-    const {width: layoutWidth, height} = this.getLayoutSize();
-    const width = layoutWidth * 0.8; // To account for 10% horizontal padding.
-    let sumDeviceWidths = 0;
+    const {
+      offsetWidth: width,
+      offsetHeight: height,
+    } = this.element.querySelector(
+      '.i-amphtml-story-dev-tools-devices-container'
+    );
+    let sumDeviceWidths = MIN_DEVICE_PADDING_PX * this.devices_.length;
```

Comment:
What is `MIN_DEVICE_PADDING_PX `?
It might be breaking this:
<img width="1628" alt="Screen Shot 2021-01-22 at 12 42 29 PM" src="https://user-images.githubusercontent.com/3860311/105525839-985a0280-5caf-11eb-95cd-94b66e8396fc.png">

Reply:
My bad, I meant to remove that but forgot to remove that usage. Sent the commit